shard.lock

```diff
@@ -18,7 +18,7 @@ shards:
 
   mqtt-protocol:
     git: https://github.com/84codes/mqtt-protocol.cr.git
-    version: 0.2.0+git.commit.5cf504b32313b3b63eecde22e2a136b9b9935b8b
+    version: 0.2.0+git.commit.739428e1cd4ea9dab1bb6cf101fb18ea54a099e5
```

Comment:
Maybe we should have updated the version of mqtt-protocol to 0.3.0? Maybe start using tags too...